packages/trader/src/AppV2/Components/OnboardingGuide/TradeParamsGuide/trade-params-guide.tsx

```diff
@@ -0,0 +1,50 @@
+import { useLocalStorageData } from '@deriv/hooks';
+import { Localize } from '@deriv/translations';
+import { getLocalStorage } from '@deriv/utils';
+import React from 'react';
+import { Step } from 'react-joyride';
+import GuideContainer from '../GuideForPages/guide-container';
+
+const TradeParamsGuide = () => {
+    const [guide_dtrader_v2, setGuideDtraderV2] = useLocalStorageData<Record<string, boolean>>('guide_dtrader_v2');
+    const [show_guide, setShowGuide] = React.useState(false);
+    const timerRef = React.useRef<NodeJS.Timeout>();
+
+    React.useEffect(() => {
+        if (!guide_dtrader_v2?.trade_params) {
+            const element = document.querySelector('.trade');
+            const observer = new MutationObserver(() => {
+                if (document.getElementsByClassName('trade__parameter-tooltip-info').length > 0) {
+                    const latest_guide_dtrader_v2 = getLocalStorage('guide_dtrader_v2');
+                    timerRef.current = setTimeout(() => {
+                        setShowGuide(true);
+                        setGuideDtraderV2({ ...latest_guide_dtrader_v2, trade_params: true });
+                    }, 300);
+                    observer.disconnect();
+                }
+            });
+            if (element) observer.observe(element, { childList: true, subtree: true });
+        }
+        return () => {
+            if (timerRef.current) {
+                clearTimeout(timerRef.current);
```

Comment:
Should we also call `disconnect()` here?

Reply:
already called inside the function above